src/nbl/video/COpenGLCommon.h

```diff
@@ -6,7 +6,7 @@
 #define __NBL_VIDEO_C_OPENGL_COMMON_H_INCLUDED__
 
 #include "BuildConfigOptions.h"
-#include "COpenGLExtensionHandler.h"
+#include "GLES3/gl32.h"
```

Comment:
what's gl32 ?

Reply:
gles3.2 header

Reply:
we're going to be using 3.1 + extensions

Reply:
you told me you want me to download gles 3.2 header so there it is

Reply:
gles extensions header is included in COpenGLESFunctionTable